.env

```diff
@@ -61,7 +61,7 @@ GCC_VERSION=""
 GO=1.19.13
 STATICCHECK=v0.4.5
 HDFS=3.2.1
-JDK=8
+JDK=17
```

Comment:
@danepitkin Does this look like a reasonable change?

Reply:
IMO we should keep this as JDK 8 so that by default CI jobs test the lowest version we support. Many jobs will override it downstream, though. e.g. https://github.com/apache/arrow/pull/38082/files